scripts/zones/Windurst_Waters/npcs/Leepe-Hoppe.lua

```diff
@@ -29,6 +29,18 @@ function onTrigger(player,npc)
 		elseif(MissionStatus >= 1 and MissionStatus < 3) then
 			player:startEvent(0x008e); -- Keep displaying the instructions
 		end
+	elseif(player:getCurrentMission(WINDURST) == AWAKENING_OF_THE_GODS and player:getVar("MissionStatus") == 0) then
+		player:startEvent(0x02DE);
+	elseif(player:getCurrentMission(WINDURST) == AWAKENING_OF_THE_GODS and player:getVar("MissionStatus") == 1) then
+		player:startEvent(0x02DF);
+	elseif(player:getCurrentMission(WINDURST) == AWAKENING_OF_THE_GODS and player:getVar("MissionStatus") == 2) then
+		player:startEvent(0x02E3);
+	elseif(player:getCurrentMission(WINDURST) == AWAKENING_OF_THE_GODS and player:getVar("MissionStatus") == 5 and player:hasKeyItem(BOOK_OF_THE_GODS)) then
+		player:startEvent(0x02E6);
+	else
+		player:startEvent(0x0159); -- Standard Dialogue?
+	end
+	---------------------------
 	elseif(player:getQuestStatus(WINDURST,FOOD_FOR_THOUGHT) == QUEST_ACCEPTED) then
 		player:startEvent(0x0137);

```

Comment:
Compare this with what you have, but most importantly, test it out. The mapserver console log will throw out errors with a line number and its best guess about what was wrong, helping you catch and correct these syntax errors.

https://www.privatepaste.com/download/990b5952d7

This is coming form a guy who's been chewed out a few times about not thoroughly testing small stuff :P